Core/Compatibility.cpp

```diff
@@ -79,6 +79,7 @@ void Compatibility::CheckSettings(IniFile &iniFile, const std::string &gameID) {
 	CheckSetting(iniFile, gameID, "ShaderColorBitmask", &flags_.ShaderColorBitmask);
 	CheckSetting(iniFile, gameID, "DisableFirstFrameReadback", &flags_.DisableFirstFrameReadback);
 	CheckSetting(iniFile, gameID, "DisableRangeCulling", &flags_.DisableRangeCulling);
+	CheckSetting(iniFile, gameID, "MpegWarmUpForGodEaterSeries", &flags_.MpegWarmUpForGodEaterSeries);
```

Comment:
Let's just call this MpegWarmUp. We might find other uses.

Reply:
I will add warm up in  sceMpegGetAtracAu for fix another games , change MpegWarmUpForGodEaterSeries to  MpegWarmUp would confuse name.

Reply:
OK, for now. I hope eventually we can find a universal solution to these instead of relying on compat.ini hacks.

Reply:
I'm obviously not a fan of the idea of having an array of warm up flags for every HLE function.  If we have to do this though, MpegAvcWarmUp would be a better name.

I think we need to figure out what the actual variables are that cause the delay, which btw is seen in most pspautotests of Mpeg even with firmware 1.50 set.

-[Unknown]

Reply:
Renamed it.

Reply:
@unknownbrackets My goal for sceMpegGetAtracAu 's warmup don't need add games in compat.ini